pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -76,3 +76,13 @@ svg {
       }
   }
 }
+
+// Let PF4 handle the scrolling through page component otherwise we might get double scrollbar
+html:not(.index-page) body {
+  overflow-y: hidden;
+
+  // Ensure UI fills the entire page (and does not run over)
+  div.pf-u-h-100 {
```

Comment:
1. This shouldn't be tied to a `<div>` element. We really should not have CSS bound to elements _and_ classes (unless necessary... but it's not necessary ~99% of the time).
2. This doesn't look like a PF class, but a CT one? It's Cockpit-specific, right? Something like ct-page-fill might be better, as it's more descriptive and signifies that it's a Cockpit-specific utility class related to the page.

Reply:
It's actually a PF class that I found through their website but I can't find it in the shipped code so I just copied their code. I can rename it of course.

Reply:
Aha. Weird that it isn't shipped. Perhaps it's new and isn't in the release of PF we're using?

Reply:
They have some stylesheets only for their website, and I think it's part of these.